test/hotspot/jtreg/serviceability/jvmti/vthread/FollowReferences/libVThreadStackRefTest.cpp

```diff
@@ -23,7 +23,7 @@
 
 #include <jni.h>
 #include <jvmti.h>
-#include <jvmti_common.h>
+#include <jvmti_common.hpp>
```

Comment:
Should this be in quotes rather than <> ?

Reply:
```suggestion
#include "jvmti_common.hpp"
```

Reply:
Not making those kinds of changes in this PR.  Also surprised this is using <atomic>.  It seems there
are a number of tests doing that.  I'll file a bug for that.
https://bugs.openjdk.org/browse/JDK-8326999